Formula/kubebuilder.rb

```diff
@@ -28,6 +29,15 @@ def install
       -X main.buildDate=#{Time.now.iso8601}
     ]
     system "go", "build", *std_go_args(ldflags: ldflags.join(" ")), "./cmd"
+
+    output = Utils.safe_popen_read("#{bin}/kubebuilder", "completion", "bash")
+    (bash_completion/"kubebuilder").write output
+
+    output = Utils.safe_popen_read("#{bin}/kubebuilder", "completion", "zsh")
+    (zsh_completion/"_kubebuilder").write output
+
+    output = Utils.safe_popen_read("#{bin}/kubebuilder", "completion", "fish")
+    (fish_completion/"kubebuilder.fish").write output
```

Comment:
```suggestion
    output = Utils.safe_popen_read(bin/"kubebuilder", "completion", "bash")
    (bash_completion/"kubebuilder").write output
    output = Utils.safe_popen_read(bin/"kubebuilder", "completion", "zsh")
    (zsh_completion/"_kubebuilder").write output
    output = Utils.safe_popen_read(bin/"kubebuilder", "completion", "fish")
    (fish_completion/"kubebuilder.fish").write output
```
Let's make this more like how it's done elsewhere. We maybe want to remove the unnecessary interpolation here too:

https://github.com/Homebrew/homebrew-core/blob/8bc285da7b0fd158e1ffaa4bf58f1dd52e5ddf6d/Formula/kubebuilder.rb#L21-L22

Squash these changes with your first commit, please.

Reply:
Thank you for review.
Fixed.